packages/provider/src/transports/wallet-request-handler.ts

```diff
@@ -379,7 +379,9 @@ export class WalletRequestHandler implements ExternalProvider, JsonRpcHandler, P
           if (typeof typedDataObject === 'string') {
             try {
               typedData = JSON.parse(typedDataObject)
-            } catch (e) {}
+            } catch (e) {
+              console.warn('error parsing typedData', e)
```

Comment:
how about updating the line to `console.warn('walletRequestHandler: error parsing typedData', e)` just to give more context?